R/utils.R

```diff
@@ -124,11 +124,11 @@ create_credentials_db <- function(db_name = credentials_name){
   
   # Init the credentials table for credentials database
   credentials <- data.frame(
-    user = "admin",
-    password = "QWERTY1",
+    user = c("ADMIN", "admin", "NONADMIN", "nonadmin"),
+    password = rep("QWERTY1", 4),
     # password will automatically be hashed
-    admin = TRUE,
-    expire = as.character(Sys.Date()),
+    admin = c(TRUE, TRUE, FALSE, FALSE),
+    expire = rep(as.character(Sys.Date()), 4),
```

Comment:
Personally I'm not a big fan of adding additional users to the initialized credential database. It is not clear to the user that multiple admin account were initialized and that they would need to open admin tools to remove the additional accounts.

![image](https://user-images.githubusercontent.com/85879620/202007461-4754a58e-f0dc-45fd-b698-3654711e2217.png)

Reply:
That's a good point. Maybe we should just switch back to a single 'admin' and 'nonadmin' account for now. Ideally, to solve the case sensitivity issue, I'd like to be able to pass a `tolower` function to all usernames entered to avoid the confusion/frusteration. I'll investigate.